src/charts/CytoViz/CytoViz.js

```diff
@@ -128,7 +157,24 @@ export const CytoViz = (props) => {
       });
     }
   };
-
+  /**
+   * appends meta data needed for the visualisatation to the data field of the edges
+   * @param {*} edges in the Cityscape format
+   * @returns the elements object with fields: {asInEdgeHighlighted: false, _asOutEdgeHighlight: false}
```

Comment:
Typo: _undesrcore_ char still present in `_asOutEdgeHighlight` :slightly_smiling_face: